modules/library/referencing/src/main/java/org/geotools/referencing/operation/matrix/XMatrix.java

```diff
@@ -112,6 +136,48 @@
     void invert() throws SingularMatrixException;
 
     /**
+     * Set to the inverse of the provided matrix.
+     * 
+     * @param matrix The matrix that is to be inverted. Not modified.
+     * @throws SingularMatrixException if this matrix is not invertible.
+     */
+    void invert( Matrix matrix ) throws SingularMatrixException;
```

Comment:
The methods were previously defined by superclasses, adding them to a common interface so we are method compatible.

Aside: this does introduce duplication with mul and multiple which I would like to address.